web/ui/templates/_base.html

```diff
@@ -40,8 +40,9 @@
             <li><a href="{{ pathPrefix }}/alerts">Alerts</a></li>
             <li><a href="{{ pathPrefix }}/graph">Graph</a></li>
             <li><a href="{{ pathPrefix }}/status">Status</a></li>
```

Comment:
How would you name the two pages? One has all the the configuration settings (flags, rules, config.yaml), the other one show the currently scraped targets and their up status. Calling them `Config` and `Status` seemed logical to me.